.github/labeler.yml

```diff
@@ -0,0 +1,193 @@
+# numpy-stubs module
+
+numpy.char.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/char/**
+
+numpy._core.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/_core/**
+
+numpy.core.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/core/**
+
+numpy.ctypeslib.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/ctypeslib.pyi
+
+numpy.distutils.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/distutils/**
+
+numpy.dtype:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/core/numerictypes.pyi
+    - src/numpy-stubs/dtypes.pyi
+
+numpy.dtypes.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/dtypes.pyi
+
+numpy.emath.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/lib/emath.pyi
+
+numpy.f2py.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/f2py/**
+
+numpy.fft.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/fft/**
+
+numpy.generic:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/_typing/numeric.pyi
+
+numpy.lib.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/lib/**
+
+numpy.linalg.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/linalg/**
+
+numpy.ma.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/ma/**
+
+numpy.matlib.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/matlib.pyi
+
+numpy.matrixlib.*:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/matrixlib/**
+
+numpy.ndarray:
+- changed-files:
+  - any-glob-to-any-file:
+    - src/numpy-stubs/core/multiarray.pyi
+    - src/numpy-stubs/_typing/array.pyi
```

Comment:
`ndarray` is (only) defined in `__init__.pyi` in the stubs. So I don't think we can automate this label.

Also; `_typing.array`? Are you using an AI-coding assistant by any chance?